### PR TITLE
Implement the function to pass data to the website

### LINKED ISSUE #27: Implement the function to transmit the value from the extension to the website.

### PR DESCRIPTION
**Pull request by HyeockJinKim**
Closes #27

### CODE REVIEW

File: background/contentScript.js [hunk 1]
```diff
@@ -10,3 +10,9 @@ window.addEventListener('message', function(event) {
     });
   }
 }, false);
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (request.from && request.from === 'popup') {
+    window.postMessage({from: 'content', data: request}, "*");
+  }
+});
```
Comment (HyeockJinKim) [inline background/contentScript.js:18]: Extension cannot directly send data from the popup screen to the website.
Therefore, it should be sent to popup -> contentscript.js -> website.

File: src/components/page/public/index.tsx [hunk 2]
```diff
@@ -30,20 +29,20 @@ export default function Page({children}: LayoutProps) {
     history.push('/register');
     currentStore.setRegister(form);
   };
-  const auth = (properties: Property[] | undefined, sign: Signature | undefined) => {
+  const auth = (host: string | undefined) => {
     history.push('/auth');
-    if (properties !== undefined) {
-      currentStore.setAuth(properties, sign);
+    if (host !== undefined) {
+      currentStore.setAuth(host);
```
Comment (HyeockJinKim) [inline src/components/page/public/index.tsx:35]: Request authentication for the property corresponding to the host address.

File: src/extension/sendToWebsite.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+export default function sendToWebsite({properties, sign, form}: MessageType) {
+  if (chrome.tabs !== undefined) {
+    chrome.tabs.query({currentWindow: true, active: true}, (tabs) => {
+      const activeTab = tabs[0];
+      chrome.tabs.sendMessage(activeTab.id!, {from: 'popup', properties, sign, form});
+    });
+  }
+}
```
Comment (HyeockJinKim) [inline src/extension/sendToWebsite.ts:17]: Send message from popup to contentscript.

**Review by HyeockJinKim — COMMENTED** [review]
self review

File: src/components/register/index.tsx [hunk 4]
```diff
@@ -1,17 +1,10 @@
...
 import {currentStore} from "../../stores";
-
-// TODO: Should move to 0auth library!
-export type DynamicFormInput = {
-  type: string;
-  label: string;
-  required: boolean;
-  name?: string;
-}
+import {defaultValue} from "@0auth/message/lib/type/defaultValue";
```
Comment (anthonyminyungi) [inline src/components/register/index.tsx:7]: Let's move the library importation to the top of the file.
```js
import React from 'react';
// giving the blank here.
import { PrivatePage } from '../common';
```

I'll fix them when I'm working on #11.

**Review by anthonyminyungi — APPROVED** [review]
Looks good. I'll go merge it.

**Pull request merged**